src/systems/force_torque/ForceTorque.cc

```diff
@@ -84,9 +93,21 @@ class gz::sim::systems::ForceTorquePrivate
   public: Entity GetLinkFromScopedName(const EntityComponentManager &_ecm,
                                        const std::string &_name,
                                        Entity _parentModel) const;
-  /// \brief Create FT sensor
-  /// \param[in] _ecm Mutable reference to ECM.
-  public: void CreateForceTorqueEntities(EntityComponentManager &_ecm);
+
+  /// \brief Create sensor
+  /// \param[in] _ecm Immutable reference to ECM.
+  /// \param[in] _entity Entity of the force-torque sensor
+  /// \param[in] _forceTorque ForceTorqueSensor component.
+  /// \param[in] _parent Parent entity component.
+  public: void AddForceTorque(
```

Comment:
You could call this `AddSensor` and rearrange the function order to reduce the diff between this sensor and others, like Imu. Not all sensors are following the same pattern, but for the ones that are, it's much easier to see their diff side-by-side.

Reply:
Good call, done. 6ef7ae9